Task1/aga/IsPrime/IsPrime/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IsPrime
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            IsPrime isPrime = new IsPrime(Convert.ToInt32(Console.ReadLine()));
+
+            isPrime.GetPrimes();
```

Comment:
Here get primes should just output an int[] array and then those should be printed to console in this main method.